plugins/modules/efs.py

```diff
@@ -767,6 +818,9 @@ def main():
                                                   throughput_mode=throughput_mode, provisioned_throughput_in_mibps=provisioned_throughput_in_mibps) or changed
         if transition_to_ia:
             changed |= connection.update_lifecycle_policy(name, transition_to_ia)
+        if backup:
+            backup = str(backup).upper()
```

Comment:
`backup` is already a string, you don't need the `str()` cast.

Reply:
Thanks for this catch